### PR TITLE
Make e2e logs easier to read by getting rid of "INFO" and double timestamps

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

**Comment by ixdy** [comment]
assigning back to you until this becomes a real PR (if it does)

**Comment by thockin** [comment]
@ixdy @quinton-hoole I think this de-noises the logs of an e2e-run quite a bit.  What do you think?

**Comment by ixdy** [comment]
LGTM

**Comment by ixdy** [comment]
wait, actually cancel LGTM.

the problem is that while in Jenkins we get timestamps, we lose them for the build log uploaded to GCS.

**Comment by thockin** [comment]
Even then, only INFO lines get timestamps - STEP lines and [AfterEach] and
so on do not.

Can we turn off Jenkins' timestamps and instead make ginkgo timestamp
everything?  what we have now is half and half and hard to follow in both
formats.
